px-context-browser.html

```diff
@@ -806,16 +982,28 @@
         }
         else if (action === 'select' || (isTerminal && action === 'activate')) {
           this.select(item, 'DOM_EVENT');
-          if(!this.keepOpen) {
+          if (!this.keepOpen) {
             this.$.dropdown.close();
           }
         }
         else if (action === 'favorite') {
-          // @TODO: Implement favorites storage/notification behavior
+          this._favoritedCop.handleItemTapped(item);
         }
       }
     },
 
+    _getFavoritedItemPath(item) {
+      if (item && typeof item === 'object' && this.favoritedMeta.item !== null && this.favoritedMeta && typeof this.favoritedMeta === 'object') {
```

Comment:
Why do we need to check this.favoritedMeta here? I don't see that we use it anywhere below, seems extraneous.

Reply:
:+1: